apps/api/v1/test/lib/selected-calendars/_post.test.ts

```diff
@@ -79,6 +79,14 @@ describe("POST /api/selected-calendars", () => {
         integration: "google",
         externalId: "ext123",
         userId: 444444,
+        id: 1,
+        eventTypeId: null,
```

Comment:
These are needed because id is required (and autometically generated by DB) and eventTypeId has default null value